File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/SaveClosedLedgerException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class SaveClosedLedgerException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1344573935915892788L;

	public SaveClosedLedgerException() {
		super("Cannot save document or modify document in open ledger.");
	}
}

File: proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/SaveClosedLedgerException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
   public class SaveClosedLedgerException : Exception
    {
       public SaveClosedLedgerException()
           : base("Cannot save document or modify document in open ledger.")
       {
       }
    }
}
